articles/hypothesis/index.md

```diff
@@ -0,0 +1,147 @@
+
+On the other hand, an alternative hypothesis results from the experiment that we hope to show. We want the alternative hypothesis to be true. It is the hypothesis that is the alternate of the null hypothesis. The image below shall aid in the understanding of these two types of hypotheses.
+
+![nullalternative](/engineering-education/hypothesis/nullalternative.png)
+
+*Null vs. alternative hypothesis*
+
+[Source](https://towardsdatascience.com/hypothesis-testing-in-machine-learning-using-python-a0dc89e169ce)
+
+#### Set a significance level
+
+After forming our null and alternative hypotheses, we should select a significance level. This is the measure of the influence of the evidence that needs to be available in a sample before rejecting the null hypothesis. The significance level is usually 5%. It means that it is probable that the test may suffer a type I error. The image below visualizes the significance level.
+
+![significancelevel](/engineering-education/hypothesis/significancelevel.png)
+
+*Alpha = 5%, denoting significance level*
+
+[Source](https://www.edugrad.com/tutorials/maths-for-machine-learning/56)
+
+Since the significance level is 5%, our level of confidence becomes 95%. This means that 95% of hypothesis tests won’t end in a type I error. You may ask why 5% and not any other value is commonly chosen. It simply is standard practice to use 5%.
+We mentioned a type I error above. Let’s define what type I and II errors are.
+**Type I error**. This is an error characterized by a scenario where we reject a true null hypothesis. The symbol alpha represents it.
+
+**Type II error**. We can define a type II error as a situation where we retain a null hypothesis, but it is false. It is denoted by beta.
+
+#### Find the region of rejection for the null hypothesis
+
+There exists a region in the sample space where we reject the null hypothesis. The rejection is if a calculated value lies in the region. This region is known as the critical region. We may reference the image in the previous section. A normal curve depicts the critical region in hypothesis testing. In the case of a type I error, it is referred to as the alpha region. It is a beta region in the context of a type II error.
+
+#### Compute p-value
+
+Assuming the null hypothesis is true, the probability of getting an outcome at least as extreme as the observed outcome of a hypothesis test is the p-value. The meaning of “extreme” is dependent on how the testing is carried out. The p-value determines whether there is enough evidence to retain the alternative hypothesis or retain the null hypothesis.
```

Comment:
There are several tests to compute the critical region. Why did you go for the p-test? z-test, f-test, and chi-squared test are other possible tests as well. In this example, the p-test makes sense as we are comparing two distributions. However, given the mean or standard deviation of the distributions, scenarios change. I think if we can just brief about that, the reader will understand the significance of these tests,